archaius2-commons-configuration/src/main/java/com/netflix/archaius/commons/CommonsToConfig.java

```diff
@@ -16,14 +16,15 @@
 package com.netflix.archaius.commons;
```

Comment:
Do we still need to support this or the delegate class is supposed to replace it?